extensions/replication/queuePopulator/BucketFileLogReader.js

```diff
@@ -7,7 +7,7 @@ class BucketFileLogReader extends LogReader {
     constructor(params) {
         const { zkClient, kafkaProducer, dmdConfig, logger } = params;
         super({ zkClient, kafkaProducer, logConsumer: null,
-                logId: `bucketFile_${dmdConfig.logName}`, logger });
+            logId: `bucketFile_${dmdConfig.logName}`, logger });
```

Comment:
I think we should disable the linter rule leading to this indent change, although I'm not sure what it's complaining about I don't see a value here and it is going to cause conflict resolution troubles (and possibly more annoyance depending on which editor people use).

Reply:
Slight problems :/
I made a post here:
https://github.com/scality/Guidelines/pull/59#issuecomment-328968923

New lint version wants specific changes, but since we are ignoring most of them, one of the problems we face is indent issues. Will talk with Rahul about this tomorrow